earthfile2llb/converter.go

```diff
@@ -2835,7 +2836,7 @@ func (c *Converter) checkAllowed(command cmdType) error {
 		case fromCmd, fromDockerfileCmd, locallyCmd, buildCmd, argCmd, letCmd, setCmd, importCmd, projectCmd, pipelineCmd:
 			return nil
 		default:
-			return errors.New("the first command has to be FROM, FROM DOCKERFILE, LOCALLY, ARG, BUILD or IMPORT")
+			return hint.Wrap(errors.New("requires a FROM, FROM DOCKERFILE, or LOCALLY"), "This command needs to run in a shell. You should be able to solve this by adding 'FROM someImage' on the line before this one.")
```

Comment:
`FROM <image>` or `FROM [image]` may be a more familiar way to denote an argument. But, this may be overly pedantic. 

https://en.wikipedia.org/wiki/Usage_message#Pattern